packages/flame/lib/src/game/game_widget/game_widget.dart

```diff
@@ -190,9 +194,6 @@ class _GameWidgetState<T extends Game> extends State<GameWidget<T>> {
       // Reset mouse cursor
       _initMouseCursor();
       addMouseCursorListener();
-
-      // Reset the loader future
-      _gameLoaderFuture = null;
```

Comment:
I think we need to keep this right? If the game on this widget changed, onLoad needs to run again

Reply:
It will, the cache is just in the game now instead